src/GeneratorHelper.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle;
+
+/**
+ * @author Sadicov Vladimir <sadikoff@gmail.com>
+ */
+class GeneratorHelper
+{
+    public function getFormFieldsPrintCode(array $formFields): string
+    {
+        $printCode = '';
+
+        foreach ($formFields as $fieldName) {
+            $printCode .= <<<PHP
+            ->add('$fieldName')
+PHP;
+        }
+
+        return $printCode;
+    }
+
+    public function getEntityFieldPrintCode($entity, $field): string
+    {
+        $printCode = $entity.'.'.$field['fieldName'];
+
+        if (in_array($field['type'], ['datetime'])) {
+            $printCode = $printCode.' ? '.$printCode.'|date(\'Y-m-d H:i:s\') : \'\'';
+        } elseif (in_array($field['type'], ['date'])) {
+            $printCode = $printCode.' ? '.$printCode.'|date(\'Y-m-d\') : \'\'';
+        } elseif (in_array($field['type'], ['time'])) {
+            $printCode = $printCode.' ? '.$printCode.'|date(\'H:i:s\') : \'\'';
+        } elseif (in_array($field['type'], ['array'])) {
+            $printCode = $printCode.' ? '.$printCode.'|join(\', \') : \'\'';
+        } elseif (in_array($field['type'], ['boolean'])) {
+            $printCode = $printCode.' ? \'Yes\' : \'No\'';
+        }
+
+        return $printCode;
+    }
+
+    public function getHead($baseLayoutExists, $title): string
+    {
+        if ($baseLayoutExists) {
+            $code = <<<TWIG
```

Comment:
If you just `return <<<TWIG`, then the second part doesn't need to live in an `else` :)

Reply:
@weaverryan how do you think, may be it will be better to inject this helper in generator with `fileManager` and hide base_layout detection?